tests/src/PatternAssertions/BasePatternAssert.php

```diff
@@ -193,4 +198,24 @@ protected function assertImage(?array $expected_image, string $selector, Crawler
     self::assertStringContainsString($expected_image['src'], $element->attr('src'));
   }
 
+  /**
+   * Asserts the badges items of the pattern.
+   *
+   * @param array $badges
+   *   The expected badges item values.
+   * @param \Symfony\Component\DomCrawler\Crawler $crawler
+   *   The DomCrawler where to check the element.
+   */
+  protected function assertBadgesElements(array $badges, Crawler $crawler): void {
+    if (empty($badges)) {
+      $this->assertElementNotExists('.badge', $crawler);
+      return;
+    }
+    $badges_items = $crawler->filter('.mt-2-5');
```

Comment:
when switching to default you might have issues (test failing that it does not find `EU financing`, this is because the **search** variant has some special parsing for the badges, so just double check in the bcl template that `.mt-2-5` is ok as a target and ignore the error

Reply:
I did a test on local, changed the variant to 'default' and check if the selector is good. 

The selector is good, there's a badges management at patter-card--variant-search,thml.twig. So at my test the selector was good but the badges come empty because we are not moving the info properly.